web/src/search/input/QueryBuilderInputRow.tsx

```diff
@@ -48,7 +53,8 @@ export const QueryBuilderInputRow: React.FunctionComponent<Props> = props => {
                     autoCapitalize="off"
                     autoComplete="off"
                     placeholder={placeholder}
-                    onChange={props.onInputChange(props.shortName)}
+                    // tslint:disable-next-line:jsx-no-lambda
+                    onChange={e => props.onInputChange[props.shortName](e, props.shortName)}
```

Comment:
This still has the same issue (as shown by disabling the lint rule). The idea was
```suggestion
                    onChange={props.onInputChange[props.shortName]}
```